loctool.js

```diff
@@ -402,28 +414,32 @@ function walk(dir, project) {
                         included = true;
                     }
                 }
-            }
-
-            if (included) {
-                logger.trace("Included.");
-                if (fs.existsSync(pathName)) {
-                    stat = fs.statSync(pathName);
-                    if (stat && stat.isDirectory()) {
-                        logger.info(pathName);
-                        walk(pathName, project);
-                    } else {
-                        if (project) {
-                            logger.info(relPath);
-                            project.addPath(relPath);
+            } else {
+                if (!settings.exclude.includes(pathName)) {
```

Comment:
This change will mean that when there is a project, no subdirectories will be walked... Also, it does not use minimatch.

This part of the change should be something like:

```
} else {
    if (mm.any(relPath, settings.exclude)) {
        included = false;
    }
}
```
And then continue with the code on line 407 `if (included)`. That way, we walk directories both inside and outside of projects.

Reply:
I updated it. and looks like `isMatch` is the latest version. instead `any` https://github.com/micromatch/micromatch/blob/master/index.js#L129